{
  "ThreatSeverity": "Moderate",
  "PublicDate": "2019-07-27T00:00:00Z",
  "Bugzilla": {
    "description": "CVE-2017-18379 kernel: out of bound access in drivers/nvme/target/fc.c",
    "id": "1735637",
    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1735637"
  },
  "Cvss": {
    "cvss_base_score": "",
    "cvss_scoring_vector": "",
    "status": ""
  },
  "Cvss3": {
    "cvss3_base_score": "7.0",
    "cvss3_scoring_vector": "CVSS:3.0/AV:L/AC:H/PR:L/UI:N/S:U/C:H/I:H/A:H",
    "status": "draft"
  },
  "Iava": "",
  "Cwe": "(CWE-787|CWE-125)",
  "Statement": "",
  "Acknowledgement": "",
  "Mitigation": "",
  "AffectedRelease": [],
  "PackageState": [
    {
      "product_name": "Red Hat Enterprise Linux 8",
      "fix_state": "Not affected",
      "package_name": "kernel",
      "cpe": "cpe:/o:redhat:enterprise_linux:8"
    },
    {
      "product_name": "Red Hat Enterprise Linux 8",
      "fix_state": "Not affected",
      "package_name": "kernel-rt",
      "cpe": "cpe:/o:redhat:enterprise_linux:8"
    }
  ],
  "Name": "CVE-2017-18379",
  "DocumentDistribution": "",
  "Details": [
    {
      "Detail": "In the Linux kernel before 4.14, an out of boundary access happened in drivers/nvme/target/fc.c."
    },
    {
      "Detail": "A flaw was found in the Linux kernelâ€™s implementation of NVME over a fiber connection. A local attacker could use this flaw to create a situation in which specially crafted requests could corrupt memory or possibly escalate privileges."
    }
  ],
  "References": []
}
